### DISCUSSION TITLE
Docker Compose EULA not accepted?

### DISCUSSION THREAD

**Discussion by sawahsawah** [itzg/docker-minecraft-server#1721] [Q&A] [answered]
I am trying to create a server using Docker Compose, but the container never seems to get going. I finally figured out how to see the logs and found that it keeps giving me the following error about the EULA:

[init] Please accept the Minecraft EULA at
[init]   https://account.mojang.com/documents/minecraft_eula
[init] by adding the following immediately after 'docker run':
[init]   -e EULA=TRUE

I can't figure out why this is happening, since I have the EULA set to true in my Docker Compose file:

  minecraft:
    image: itzg/minecraft-server:latest
    ports:
      - 25565:25565
    container_name: minecraft
    environment:
      - EULA= TRUE
      - SERVER_NAME= "Corn Comrades Miners"
      - MODE= "survival"
      - ONLINE_MODE= "FALSE"
    restart: unless-stopped
    volumes:
      - ./volumes/minecraft/data:/data
      - ./volumes/minecraft/mods:/mods

I have tried all variations of EULA=TRUE, EULA= TRUE, and EULA= "TRUE" but nothing seems to work. I also tried taking out the ONLINE_MODE= "FALSE" in case that was interfering with the initial setup, but no dice.

Apologies if this is a stupid mistake on my part, but thanks for any help!

**Comment by itzg** [comment, marked as answer]
The space after the equal sign should not be there. 

In general I recommend avoiding the list-style and use object style instead:

https://github.com/itzg/docker-minecraft-server/blob/master/examples/docker-compose.yml

The next problem you'll hit is the literal inclusion of quotes on those other variables.

**Reply by sawahsawah** [reply]
That did the trick! Thanks for helping me through my syntax issues :)
